app/controllers/movie_preference_controller.py

```diff
@@ -27,11 +27,25 @@ def get(self, user_id):
         if user is not User.query.get(user_id):
             abort(403)
 
-        movie_ids = [mp.external_movie_id for mp in user.movies]
+        user_movie_preferences = user.movies
 
         movies = []
-        for id in movie_ids:
-            movie = get_movie(id)
-            movies.append(movie)
+        for mp in user_movie_preferences:
```

Comment:
Changed this to merge the two dicts together; it let us simplify things on the frontend. Now, as we need things other than the title, we can just add it to the update dict here.